streamer/streamer.py

```diff
@@ -233,7 +231,11 @@ def make_out_prefix(self, input_fname, dest_dir):
 
         out_dir = pjoin(dest_dir,  self.dest_template.format(x=x_index, y=y_index, **time_dict))
         if not exists(out_dir):
-            os.makedirs(out_dir)
+            try:
+                os.makedirs(out_dir)
+            except:
+                LOG.info("Dir exsits")
```

Comment:
Could simplify these 5 lines down to `os.makedirs(out_dir, exist_ok=True)`

Reply:
Could be...patched that on my phone. Didn't think about it...tbh